application/controllers/TodoController.php

```diff
@@ -11,7 +11,7 @@ public function __construct() {
 	public function index()
 	{
 		$this->modules[]      = 'todo';
-		$this->js_listeners[] = 'Todo.fn.add_product()';
+		$this->js_listeners[] = 'Todo.fn.add_product(), Todo.table.view_table()';
```

Comment:
separate this into 2 lines `$this->js_listeners[] = 'Todo.fn.add_product()';
$this->js_listeners[] = 'Todo.table.view_tables()';`